build/charts/antrea/values.yaml

```diff
@@ -104,6 +104,8 @@ egress:
   # useful when the Node network restricts the number of secondary IPs a Node
   # can have, e.g. EKS. It must not be greater than 255.
   maxEgressIPsPerNode: 255
+  # -- Enable Egress traffic shaping ability.
```

Comment:
s/ability/capability

or you can even remove "ability" altogether

Reply:
Done.